src/PackageDependencyProvider.ts

```diff
@@ -118,7 +118,7 @@ export class PackageDependenciesProvider implements vscode.TreeDataProvider<Tree
         // Refresh the tree when a package resolve or package update task completes.
         vscode.tasks.onDidEndTask((event) => {
             const definition = event.execution.task.definition;
-            if (definition.command === 'swift' && definition.args[0] === 'package' &&
+            if (definition.type === 'swift' && definition.args[0] === 'package' &&
```

Comment:
I changed this from command to type because command will include the swift path if it is set. Personally I think we should be recognising resolve, update tasks by their name not their command line. That is how vscode does it.

Reply:
LLDB sessions are initiated via the launch.json. They use the task name to call the build tasks